chain/jsonrpc/jsonrpc-tests/tests/rpc_query.rs

```diff
@@ -10,14 +10,16 @@ use near_jsonrpc::client::{new_client, ChunkId};
 use near_jsonrpc_primitives::types::query::QueryResponseKind;
 use near_jsonrpc_primitives::types::validator::RpcValidatorsOrderedRequest;
 use near_logger_utils::init_test_logger;
-use near_network::test_utils::WaitOrTimeoutActor;
+use near_network::test_utils::wait_or_timeout;
 use near_primitives::account::{AccessKey, AccessKeyPermission};
 use near_primitives::hash::CryptoHash;
 use near_primitives::types::{BlockId, BlockReference, EpochId, SyncCheckpoint};
 use near_primitives::views::QueryRequest;
 
 use near_jsonrpc_tests::{self as test_utils, test_with_client};
 
+pub type ControlFlow = std::ops::ControlFlow<()>;
+
```

Comment:
```suggestion
```

I don't think this type alias is needed, just a `use` seems to work fine?

Reply:
I don't have any preference. I just followed the model PR. The `ControlFlow` is defined as type in runner.rs. 

https://github.com/near/nearcore/blob/7efb57a06906b297937fc0596139bd3da85aafc7/integration-tests/src/tests/network/runner.rs#L36

I am updating this PR with your suggestion though.

Reply:
Fixed